contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -389,6 +401,26 @@ contract StakingVault is IStakingVault, IBeaconProxy, BeaconChainDepositLogistic
         emit Reported(_valuation, _inOutDelta, _locked);
     }
 
+    /**
+     * @notice Pauses deposits to beacon chain
+     * @dev Can only be called by the vault owner
+     */
+    function pauseBeaconDeposits() external onlyOwner {
```

Comment:
Should we do some other checks, like in PausableUntil? 
`Revert if already paused` e.g.

Reply:
Fixed in https://github.com/lidofinance/core/pull/922/commits/ffcb109ff7d65d21ef5cf0f31bffb652e878397c